hack/codegen.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
+ARTIFACTS=${ARTIFACTS:-/tmp}
```

Comment:
Double quotes around curly brackets?

Reply:
I can add that indeed

Reply:
Do we need double quotes here? Check [this](https://github.com/metal3-io/cluster-api-provider-metal3/blob/036ea1866c391e8d22213cfc7c0ebba597292b1d/hack/codegen.sh#L5)